loader/src/main/java/eu/europeana/fulltext/loader/service/XMLParserService.java

```diff
@@ -434,28 +454,59 @@ private void parseAnnotationTarget(StartElement targetElement, AnnoPage annoPage
 
         // set target
         if (urlAndCoordinates.length > 1) {
-            Target t = createTarget(urlAndCoordinates[1]);
+            Target t = createTarget(urlAndCoordinates[1], anno.isMedia());
             if (anno.getTgs() == null) {
                 anno.setTgs(new ArrayList<>());
             }
             anno.getTgs().add(t);
         }
     }
 
-    private Target createTarget(String coordinates) throws LoaderException {
+    private Target createTarget(String coordinates, boolean isMedia) throws LoaderException, ParseException {
+
         String[] separatedCoordinates = coordinates.split(",");
-        if (separatedCoordinates.length != 4) {
-            throw new IllegalValueException("target '" + coordinates +
-                    "' doesn't have 4 integers separated with a comma");
+
+        if (isMedia){
+            if (separatedCoordinates.length != 2) {
+                throw new IllegalValueException(TARGET + coordinates +  "' must contain 2 NormalPlayTime-formatted " +
+                                                "parameters for start and end time, separated with a comma");
+            }
+
+            NormalPlayTime nptStart = NormalPlayTime.parse(checkNPTFormat(separatedCoordinates[0]));
```

Comment:
Ah, found it. I feel we should catch the ParseException here and rethrow as IllegalValueException like e do in the else part below

Reply:
That's indeed what I did.